package.json

```diff
@@ -17,13 +17,14 @@
   "license": "Apache-2.0",
   "scripts": {
     "test": "karma start",
-    "cover": "grunt hint && grunt jscs && npm test -- --single-run",
-    "postinstall": "npm install -g grunt-cli"
```

Comment:
I'm all for dropping the global postinstall, but we still need grunt.
Could add it Travis scripts and make it required to setup dev env?

Reply:
I'll add it back. I forgot to undo.